cwltool/stdfsaccess.py

```diff
@@ -44,3 +49,9 @@ def join(self, path, *paths):  # type: (Text, *Text) -> Text
 
     def realpath(self, path):  # type: (Text) -> Text
         return os.path.realpath(path)
+
+    # On windows os.path.realpath appends unecessary Drive, here we would avoid that
+    def docker_compatible_realpath(self, path):  # type: (Text) -> Text
+        if onWindows():
+            return path
```

Comment:
Actually, on windows Drive letter gets appended. The E drive in docker path, this was the cause of it. One thing We can do is if we are on windows calculate os.path.realpath then trim the drive letter and and again change path separater to `/` as this path goes in docker path. Should I do that